cmd/skopeo/utils.go

```diff
@@ -16,8 +17,18 @@ type errorShouldDisplayUsage struct {
 	error
 }
 
+// Check if container-storage are used before doing the maybeReexec
 func needsRexec(c *cli.Context) error {
-	return maybeReexec()
+	for _, arg := range c.Args() {
+		parts := strings.SplitN(arg, ":", 2)
+		if len(parts) != 2 {
```

Comment:
If we are going to do this, let’s add a `c/image/transports.TransportFromImageName` (which returns a `types.ImageTransport`) instead of hard-coding it in skopeo.